src/aktualizr_secondary/opcuaserver_secondary_delegate.h

```diff
@@ -11,15 +11,15 @@ class AktualizrSecondaryCommon;
 
 class OpcuaServerSecondaryDelegate : public opcuabridge::ServerDelegate {
  public:
-  OpcuaServerSecondaryDelegate(AktualizrSecondaryCommon*);
+  explicit OpcuaServerSecondaryDelegate(AktualizrSecondaryCommon* /*secondary*/);
 
-  void handleServerInitialized(opcuabridge::ServerModel*) override;
-  void handleVersionReportRequested(opcuabridge::ServerModel*) override;
-  void handleMetaDataFileReceived(opcuabridge::ServerModel*) override;
-  void handleAllMetaDataFilesReceived(opcuabridge::ServerModel*) override;
-  void handleDirectoryFilesSynchronized(opcuabridge::ServerModel*) override;
-  void handleOriginalManifestRequested(opcuabridge::ServerModel*) override;
-  void handleDirectoryFileListRequested(opcuabridge::ServerModel*) override;
+  void handleServerInitialized(opcuabridge::ServerModel* /*unused*/) override;
+  void handleVersionReportRequested(opcuabridge::ServerModel* /*unused*/) override;
+  void handleMetaDataFileReceived(opcuabridge::ServerModel* /*unused*/) override;
+  void handleAllMetaDataFilesReceived(opcuabridge::ServerModel* /*unused*/) override;
+  void handleDirectoryFilesSynchronized(opcuabridge::ServerModel* /*unused*/) override;
+  void handleOriginalManifestRequested(opcuabridge::ServerModel* /*unused*/) override;
+  void handleDirectoryFileListRequested(opcuabridge::ServerModel* /*unused*/) override;
```

Comment:
I don't understand these `unused` comments. The variables in question look pretty well used to me. (This isn't critical and shouldn't hold up the PR.)

Reply:
That's mean that the variables in declarations is never used. Only in implementation, but I think we could remove this check and restore old names

Reply:
Not a big deal on this one, it's just confusing, since it the `unused` comment makes it seem like the variable is unused in the implementation. I know the names can be left out of the declaration, but it would probably be best if they matched the definition name (in this case, `model`).